### ISSUE TITLE
support v1 KubeSchedulerConfiguration

### ISSUE THREAD

**Issue by sanposhiho** [kubernetes-sigs/kube-scheduler-simulator#46]
Currently, the simulator supports v1beta2

https://github.com/kubernetes/enhancements/issues/2891

/kind feature

**Comment by sanposhiho** [comment]
At least, the version needs to be explicitly stated. Currently, users cannot see what's version KubeSchedulerConfiguration is.

![Screen Shot 2021-12-07 at 15 27 59](https://user-images.githubusercontent.com/44139130/144977961-51d7a575-8e26-4d5b-b116-2793e833af2f.png)

**Comment by markstur** [comment]
/assign
I'll look at this next.  I was already showing the kind/apiVersion, but want to look into supporting multiple versions.

**Comment by sanposhiho** [comment]
/remove-lifecycle stale

/triage accepted
Not to mark as stale.

**Comment by markstur** [comment]
Thank you @sanposhiho .  Sorry for the delay. I made progress on this but have been working on other things. Will update soon.

**Comment by sanposhiho** [comment]
No worries :)

**Comment by sanposhiho** [comment]
@markstur Would you mind my taking over this? 
It blocks the version update.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/issues/221

**Comment by markstur** [comment]
Hello @sanposhiho 

I have an update.  I had to rebase/update and figured out that the vendoring, submodule, test updates, etc should be a separate PR before introducing all the config+multipoint changes.  So I just pushed PR and tagged you ( #222 ).  I made that "Part of" this issue, but we should do version updates separately before changing the config and functionality.

Next my multipoint code isn't 100%.  I will take a look at sharing it with you, but I've been blocking this too long so...

YES you can take over. Thank you.

**Comment by sanposhiho** [comment]
Thanks @markstur, I'll review #222 soon. 👍

**Comment by sanposhiho** [comment]
btw, I thought v1beta2 component config was completely removed from k/k at v1.24 😓  (That's why I said this issue is blocking the version update.)

**Comment by sanposhiho** [comment]
So, please feel free to continue to work on this if you like. 
I understood that it's actually not blocking version updates there was no need to rush to work on it.

**Comment by sanposhiho** [comment]
yeah, we should support v1.
/retitle support v1 KubeSchedulerConfiguration

**Comment by sanposhiho** [comment]
memo: #230 needs to be done before it so that we can simply handle plugins registered in multi point.
Once #230 is done, we can register all kind of plugins as wrapped plugins regardless of which extension point each plugin belongs to.

**Comment by sanposhiho** [comment]
It's already done by #276
